cli/src/commands/dsl/validate.ts

```diff
@@ -34,6 +35,33 @@ export default class Validate extends BaseCommand {
     };
   }
 
+  private async getTSConfig() {
+    let TSEnvConfig, configFile;
+
+    const EnvTSConfig = glob.sync("**/tsconfig.json")[0];
```

Comment:
Do we want to grab any `tsconfig.json` file or should we just grab one from the root? What if that config extends from another config?